### PR TITLE
Mynewt: Update workspace for new TCLR (PTS 8 update)

### PR DESCRIPTION
**Pull request by KKopyscinski**

### CODE REVIEW

**Comment by KKopyscinski** [comment]
removed mesh models, fixed typo in commit name

File: workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<WORKSPACE_INFORMATION NAME="Mynewt Nimble Host" CREATED_WITH="Profile Tuning Suite, 7.6.0, 15" BD_ADDR="000000000000" QDID="127651">
+<WORKSPACE_INFORMATION NAME="Mynewt Nimble" PATH="C:\Users\krzys\Desktop\Mynewt Nimble" CREATED_WITH="Profile Tuning Suite, 8.0.0, 29" BD_ADDR="000000000000" QDID="164784">
```
Comment (michal-narajowski) [inline workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6:1]: Remove path

**Comment by KKopyscinski** [comment]
Fixed consistency checks

**Comment by KKopyscinski** [comment]
Updated with supporf of ECFC, removed interoperability tests

**Comment by KKopyscinski** [comment]
Added TSPC_GAP_25_11 and TSPC_GAP_25_12, updated workspace to PTS 8.0.1

File: workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6 [hunk 2]
```diff
@@ -3435,13 +3541,13 @@
           <Row>
             <Name>TSPC_MESH_0a_2</Name>
             <Description>Mesh v1.0.1 (C2)</Description>
-            <Value>TRUE</Value>
+            <Value>FALSE</Value>
```
Comment (michal-narajowski) [inline workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6:3544]: I think we should support it.

Comment (KKopyscinski) [reply workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6]: Ok, now it's updated

**Review by michal-narajowski — APPROVED** [review]
LGTM

**Pull request merged**